tests/integration/test_sqlalchemy_integration.py

```diff
@@ -112,143 +118,159 @@ def test_insert(trino_connection):
-    conn.execute(ins, [
-        {"id": 2, "name": "wendy", "fullname": "Wendy Williams"},
-        {"id": 3, "name": "john", "fullname": "John Doe"},
-        {"id": 4, "name": "mary", "fullname": "Mary Hopkins"},
-    ])
+    conn.execute(
+        ins,
+        [
+            {"id": 2, "name": "wendy", "fullname": "Wendy Williams"},
+            {"id": 3, "name": "john", "fullname": "John Doe"},
+            {"id": 4, "name": "mary", "fullname": "Mary Hopkins"},
+        ],
+    )
     query = sqla.select(users)
     result = conn.execute(query)
     rows = result.fetchall()
     assert len(rows) == 3
-    assert frozenset(rows) == frozenset([
-        (2, "wendy", "Wendy Williams"),
-        (3, "john", "John Doe"),
-        (4, "mary", "Mary Hopkins"),
-    ])
+    assert frozenset(rows) == frozenset(
+        [
+            (2, "wendy", "Wendy Williams"),
+            (3, "john", "John Doe"),
+            (4, "mary", "Mary Hopkins"),
+        ]
+    )
     metadata.drop_all(engine)
 
 
-@pytest.mark.parametrize('trino_connection', ['tpch'], indirect=True)
+@pytest.mark.parametrize("trino_connection", ["tpch"], indirect=True)
 def test_operators(trino_connection):
     _, conn = trino_connection
     metadata = sqla.MetaData()
-    customers = sqla.Table('nation', metadata, schema='tiny', autoload_with=conn)
+    customers = sqla.Table("nation", metadata, schema="tiny", autoload_with=conn)
     query = sqla.select(customers).where(customers.c.nationkey == 2)
     result = conn.execute(query)
     rows = result.fetchall()
     assert len(rows) == 1
     for row in rows:
-        assert isinstance(row['nationkey'], int)
-        assert isinstance(row['name'], str)
-        assert isinstance(row['regionkey'], int)
-        assert isinstance(row['comment'], str)
+        assert isinstance(row["nationkey"], int)
+        assert isinstance(row["name"], str)
+        assert isinstance(row["regionkey"], int)
+        assert isinstance(row["comment"], str)
 
 
-@pytest.mark.parametrize('trino_connection', ['tpch'], indirect=True)
+@pytest.mark.parametrize("trino_connection", ["tpch"], indirect=True)
 def test_conjunctions(trino_connection):
     _, conn = trino_connection
     metadata = sqla.MetaData()
-    customers = sqla.Table('customer', metadata, schema='tiny', autoload_with=conn)
-    query = sqla.select(customers).where(and_(
-        customers.c.name.like('%12%'),
-        customers.c.nationkey == 15,
-        or_(
-            customers.c.mktsegment == 'AUTOMOBILE',
-            customers.c.mktsegment == 'HOUSEHOLD'
-        ),
-        not_(customers.c.acctbal < 0)))
+    customers = sqla.Table("customer", metadata, schema="tiny", autoload_with=conn)
+    query = sqla.select(customers).where(
+        and_(
+            customers.c.name.like("%12%"),
+            customers.c.nationkey == 15,
+            or_(customers.c.mktsegment == "AUTOMOBILE", customers.c.mktsegment == "HOUSEHOLD"),
+            not_(customers.c.acctbal < 0),
```

Comment:
Does black reformat this away?

```suggestion
        and_(
            customers.c.name.like("%12%"),
            customers.c.nationkey == 15,
        or_(customers.c.mktsegment == "AUTOMOBILE", customers.c.mktsegment == "HOUSEHOLD"),
        not_(customers.c.acctbal < 0),
```

Reply:
Actually the `_or(...)` is  an argument to the `_and` function, so this seems correct.